core/runner/src/test/java/com/cognifide/aet/runner/processing/steps/StepManagerTest.java

```diff
@@ -96,7 +96,7 @@ public void setUp() throws Exception {
     when(suiteIndexWrapper.get()).thenReturn(suite);
     when(suite.getCorrelationId()).thenReturn(CORRELATION_ID);
     when(runnerConfiguration.getMttl()).thenReturn(100L);
-    when(runnerConfiguration.getUrlPackageSize()).thenReturn(2);
+    when(runnerConfiguration.getUrlPackageSize()).thenReturn(1);
```

Comment:
This value shouldn't change. This unit test (actually this is abstract class so the test that inherits this class) checks if it is possible to set an `urlPackageSize`. And value `2` is perfectly fine here. As you will notice this will probably also fail unit tests and Travis build.

Reply:
Fixed